app/src/main/java/org/eyeseetea/malariacare/domain/entity/Option.java

```diff
@@ -132,23 +113,14 @@ public String toString() {
         public Attribute(long id, String backgroundColour,
                 HorizontalAlignment horizontalAlignment,
                 VerticalAlignment verticalAlignment, int textSize) {
-            this.id = id;
-            this.backgroundColour = backgroundColour;
-            this.horizontalAlignment = horizontalAlignment;
-            this.verticalAlignment = verticalAlignment;
-            this.textSize = textSize;
-        }
 
-        public Attribute() {
+            setId(id);
```

Comment:
Can you remove private setters? It's not necessary we have private setters, so we reduce the size of the class